### ISSUE TITLE
Ensure Android's accessibility scanner and the play store's accessibility report works with Flutter

### ISSUE THREAD

**Issue by matthew-carroll** [flutter/flutter#39531]
b/291786021

Ensure Android's accessibility scanner and the play store's accessibility report works with Flutter.

**Comment by 12people** [comment]
It's worth noting that Flutter's own docs recommend Accessibility Scanner: https://docs.flutter.dev/development/accessibility-and-localization/accessibility

Until the Scanner starts supporting Flutter apps, this mention should be removed.

**Comment by shababhsiddique** [comment]
Facing this issue in 2023. Is this really unfixable? 
![image](https://user-images.githubusercontent.com/7587873/221133633-adf48e7c-0963-4763-a638-aa2a3ce92ae7.png)

**Comment by devfemibadmus** [comment]
I had similar issue I uses sematic label but still the content labeling
number I creases and sometimes decrease in more release all I do is, I just
ignore and push to production in 2 days my app is available to download on
store

**Comment by lisovyk** [comment]
Much needed – I get hundreds of warnings on every push to app store, would be easy to miss something important.

**Comment by devfemibadmus** [comment]
all we do is ignore it all

**Comment by starchykov** [comment]
In the Flutter 3.10 release, this mistake still exists. After looking into it, I couldn't find a way to fix this warning. It’s up to Team Flutter. 

![Screenshot_1684948468](https://github.com/flutter/flutter/assets/48174604/167ca8fb-0ba9-47b1-a784-6ba90d5f3dca)

**Comment by devfemibadmus** [comment]
Yeah, I hope in the next few version this issue is fixed but lol 😂😂😂
funny bcus it’s supposed to be test on play store

**Comment by Thalinda** [comment]
Still the same in July of 2023 
![image](https://github.com/flutter/flutter/assets/45384206/d0f5c715-ea6c-4009-8aae-40cdc3311f82)

**Comment by chunhtai** [comment]
cl/555396437 to skip flutter app for play store report
